ant-networking/src/cmd.rs

```diff
@@ -153,6 +153,15 @@ pub enum LocalSwarmCmd {
     AddNetworkDensitySample {
         distance: Distance,
     },
+    /// Send peer scores (collected from storage challenge) to replication_fetcher
+    NotifyPeerScores {
+        peer_scores: Vec<(PeerId, bool)>,
```

Comment:
Does that send our local peer scores to other peers? I'm not sure this should be shared as attackers can easily make these up.

Reply:
no. `replication_fetcher` is just a container held by the smart_driver instance inside the node.
`peer_scores` is only held within a node, never shared out